### ISSUE TITLE
Does @angular-eslint Work by Default, or Do I Need to Specify the Rules?

### ISSUE THREAD

**Issue by jon9090** [angular-eslint/angular-eslint#1925]
I created an Nx with Angular project and noticed it installs @angular-eslint:

```json
"@angular-eslint/eslint-plugin": "^18.0.1",
"@angular-eslint/eslint-plugin-template": "^18.0.1",
"@angular-eslint/template-parser": "^18.0.1",
```

I added the following in my app.component:

```html
<img src="foo">
```

I expected this to trigger a lint warning based on the `@angular-eslint/template/alt-text` rule (https://github.com/angular-eslint/angular-eslint/blob/main/packages/eslint-plugin-template/docs/rules/alt-text.md).

Does @angular-eslint work by default, or do I need to specify the rules? If so, how do I do that?

**Comment by JamesHenry** [comment]
@jon9090 This is more a question of how to configure eslint, for which the official documentation is the best resource. Additionally, angular-eslint itself is not responsible for the config generated by Nx, so queries about the config should be reported on the Nx repo.

With that said `@nx/angular-template` is what you will see in your project level `.eslintrc.json` files generated for you by Nx. Nx is open source so you can see what it sets for you in terms of rules: https://github.com/nrwl/nx/blob/e5d3f9029898953ce1536cf9d6794441f7bf3a99/packages/eslint-plugin/src/configs/angular-template.ts#L17

It extends from the recommended set for `@angular-eslint/eslint-plugin-template`: https://github.com/angular-eslint/angular-eslint/blob/c8696d313260035ac2566344c1351eb5abf1ab3f/packages/eslint-plugin-template/src/configs/recommended.json#L1

So, no, it does not set that rule, by default.

You can either apply the rule to your _project_ level `.eslintrc.json` directly (within the `*.html` override):

```json
"@angular-eslint/template/alt-text": "error",
```

...or extend from the accessibility config

```json
extends: ['plugin:@angular-eslint/template/accessibility'],
```

...again, within the `*.html` override.

Hope that helps!
